### DISCUSSION TITLE
Are there any plans to allow for various oauth-type providers for Authentication to DevLake rather than the users table?

### DISCUSSION THREAD

**Discussion by rhanton** [apache/devlake#7361] [Q&A]
While what exists for team configurations at https://devlake.apache.org/docs/Configuration/TeamConfiguration/ isn't bad, are there any plans that I'm not seeing on the roadmap to help us to onboard larger teams of people quickly via our in-house staff oauth identity provider?

**Comment by klesh** [comment]
Hi, I am not sure if I understand your question.
Could you file a Feature Request and describe your situation in detail and the solution you prefer?
Thanks
